Trigger.md

```diff
@@ -0,0 +1,99 @@
+|avroSchema|Avro schema for generic record deserialization|
+
+For connection to a secure Kafka Broker -
+
+|Authentication Setting|librdkafka property|Description|
+|-|-|-|
+|authenticationMode|sasl.mechanism|SASL mechanism to use for authentication|
+|username|sasl.username|SASL username for use with the PLAIN and SASL-SCRAM|
+|password|sasl.password|SASL password for use with the PLAIN and SASL-SCRAM|
+|protocol|security.protocol|Security protocol used to communicate with brokers|
+|sslKeyLocation|ssl.key.location|Path to client's private key (PEM) used for authentication|
+|sslKeyPassword|ssl.key.password|Password for client's certificate|
+|sslCertificateLocation|ssl.certificate.location|Path to client's certificate|
+|sslCaLocation|ssl.ca.location|Path to CA certificate file for verifying the broker's certificate|
+
+# JS/TS/PS/Python Configuration
+|function.json property|Descirption|
+|-|-|
+|type|Must be set to kafkaTrigger.|
+|direction|Must be set to in.|
+|name|Name of the variable that represents  request or request body in the function code.|
+|brokerList|Defines the broker list.|
+|cardinality|Cardinality of the trigger input. Choose 'One' if the input is a single message or 'Many' if the input is an array of messages. If you choose 'Many', please set a dataType. Default: 'One'|
+|dataType|<p>Defines how Functions runtime should treat the parameter value. Possible values are:</p><ul><li>""(Default): Get the value as a string, and try to deserialize to actual parameter type like POJO.</li><li>string: Always get the value as a string</li><li>binary: Get the value as a binary data, and try to deserialize to actual parameter type byte[].</li></ul>|
+
+For connection to a secure Kafka Broker -
+
+|function.json property|librdkafka property|Description|
+|-|-|-|
+|authenticationMode|sasl.mechanism|SASL mechanism to use for authentication|
+|username|sasl.username|SASL username for use with the PLAIN and SASL-SCRAM|
+|password|sasl.password|SASL password for use with the PLAIN and SASL-SCRAM|
+|protocol|security.protocol|Security protocol used to communicate with brokers|
+|sslKeyLocation|ssl.key.location|Path to client's private key (PEM) used for authentication|
+|sslKeyPassword|ssl.key.password|Password for client's certificate|
+|sslCertificateLocation|ssl.certificate.location|Path to client's certificate|
+|sslCaLocation|ssl.ca.location|Path to CA certificate file for verifying the broker's certificate|
+
+ When you are developing locally, add your application settings in the local.settings.json file in the Values collection.
+
+**NOTE:** Username and password should reference a Azure function configuration variable and not be hardcoded.
+
+
+# host.json settings
```

Comment:
Great to view host.json, could you please explore settings for triggers and test them on your local if they are working and we can add them in public docs
https://github.com/Azure/azure-functions-kafka-extension/blob/dev/src/Microsoft.Azure.WebJobs.Extensions.Kafka/Config/KafkaOptions.cs